cloud-sql/mysql/activerecord/README.md

```diff
@@ -127,17 +122,22 @@ Navigate towards `http://localhost:3000` to verify your application is running c
 
 ## Deploy to Google App Engine Standard
```

Comment:
Do we want Cloud Run instructions too?

Reply:
Yes indeed! But I'd like to keep this PR focused on the sample app update. We can address expanding the README instructions in a follow-up PR.